tests/tools/s3/__init__.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
+
+import json
+import os
+import pathlib
+
+from .utils import S3ResourceFetcher
+
+
+def s3_download(resource_type, resource_name, resource_tags="{}", path=None, first=False):
```

Comment:
Since this is the real API function, this should add a documentation. Or it should refer to the documentation of underlying `download` function. 

This should definitely add a doc line saying - Any exception raised are not handled, so it's a responsibility of the caller to handle the exception.

Reply:
That sounds like a good idea. I'll add it.